libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -342,17 +343,24 @@ public void updateLocationLayer(Location location) {
    */
   public void startNavigation(NavigationViewOptions options) {
     // Initialize navigation with options from NavigationViewOptions
-    navigationViewModel.initializeNavigationOptions(getContext().getApplicationContext(),
-      options.navigationOptions().toBuilder().isFromNavigationUi(true).build());
-    // Initialize the camera (listens to MapboxNavigation)
-    initCamera();
-
-    setupListeners(options);
-
-    locationViewModel.updateShouldSimulateRoute(options.shouldSimulateRoute());
-    routeViewModel.extractRouteOptions(options);
-    // Everything is setup, subscribe to the view models
-    subscribeViewModels();
+    if (!isInitialized) {
+      navigationViewModel.initializeNavigationOptions(getContext().getApplicationContext(),
+        options.navigationOptions().toBuilder().isFromNavigationUi(true).build());
+      // Initialize the camera (listens to MapboxNavigation)
+      initCamera();
+      setupListeners(options);
+      // Update the view models
+      locationViewModel.updateShouldSimulateRoute(options.shouldSimulateRoute());
```

Comment:
Do these two lines have to be in this order? Could they go before/after the `if/else`, since they are in both blocks